src/components/Pages/SignupForm/signupform.module.scss

```diff
@@ -64,4 +57,26 @@ input::placeholder {
     position: absolute;
```

Comment:
there are two position properties here - never duplicate properties, as this is bad for code quality. Always double check to see if you have this property in css already